config/deploy/olm/catalogsource.yaml

```diff
@@ -6,4 +6,6 @@ spec:
   sourceType: grpc
   image: quay.io/kuadrant/kuadrant-operator-catalog:latest
   displayName: Kuadrant Operators
+  grpcPodConfig:
+    securityContextConfig: restricted
```

Comment:
Is this something we need to override in OLM for cert manager to work? What does the `securityContextConfig` adds to the grpc Pod?